pkgs/development/python-modules/zha-quirks/default.nix

```
@@ -1,13 +1,14 @@
 { stdenv, buildPythonPackage, fetchPypi
-, aiohttp, zigpy
+, aiohttp, zigpy, conftest, asynctest
 , pytest }:
 
 buildPythonPackage rec {
   pname = "zha-quirks";
   version = "0.0.45";
 
-  nativeBuildInputs = [ pytest ];
+  #nativeBuildInputs = [ pytest ];
   buildInputs = [ aiohttp zigpy ];
```

these are python packages, should be propagatedBuildInputs
```suggestion
  propagatedBuildInputs = [ aiohttp zigpy ];
```